datatypes/src/iosTest/kotlin/Test.kt

```diff
@@ -0,0 +1,9 @@
+import io.kotest.core.spec.style.FreeSpec
+import io.kotest.matchers.shouldNotBe
+
+class Test: FreeSpec( {
+
+    "This dummy test" {
+       "is just making shure" shouldNotBe "that iOS tests are indeed running"
```

Comment:
I see, you did even include a typo to make sure it's really your test that runs!

Reply:
https://www.shure.com/